modules/hcp-custom-cert.adoc

```diff
@@ -66,6 +63,6 @@ $ oc apply -f <hosted_cluster_config>.yaml
 
 * Check the API server pods to ensure that the new certificate is mounted.
 
-* Test the conncetion to the API server by using the custom domain name.
+* Test the connection to the API server by using the custom domain name.
 
 * Verify the certificate details in your browser or by using tools such as `openssl`.
```

Comment:
Do we need to add specific instructions for how to verify the certificate details?